modular_skyrat/modules/cargo/code/items/AFAD.dm

```diff
@@ -0,0 +1,18 @@
+/obj/item/gun/medbeam/afad
+	name = "Automated First Aid Device"
+	desc = "Usually supplied in medkits, the AFAD is a revolutionary device meant for fixing scrapes and bruises, and totally not a knockoff of the legendary medibeam. A label on the underside reminds you not to cross the beams."
+	icon = 'icons/obj/chronos.dmi'
+	icon_state = "chronogun"
+	inhand_icon_state = "chronogun"
```

Comment:
Could we have pictures of what it looks like in-game?

Reply:
![image](https://user-images.githubusercontent.com/62520989/181624985-827a825e-f661-4cf6-b402-ebc0e95296c7.png)
it looks like a medbeam